### ISSUE TITLE
Accessibility: Headings and Labels (WCAG 2.4.6)

### ISSUE THREAD

**Issue by veler** [Microsoft/pxt#2363]
Standard:
Labeling forms and other objects description. But also heading the page to let the user understanding what is the purpose of the current page.
http://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-descriptive.html

Current situation:
It seems there is no information that let the user understand what is the goal of the main page of MakeCode (the page with the editor and simulator). So the user can arrives on the page, but will potentially need time to understand what is the global goal of the page. One or several hidden head tag(s) could potentially fix this lack of information.
